ci-operator/step-registry/origin/e2e/test/origin-e2e-test-commands.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+cluster_profile=/var/run/secrets/ci.openshift.io/cluster-profile
+export AWS_SHARED_CREDENTIALS_FILE=${cluster_profile}/.awscred
+export AZURE_AUTH_LOCATION=${cluster_profile}/osServicePrincipal.json
+export GCP_SHARED_CREDENTIALS_FILE=${cluster_profile}/gce.json
+export HOME=/tmp/home
+export PATH=/usr/libexec/origin:$PATH
+
+trap 'CHILDREN=$(jobs -p); if test -n "${CHILDREN}"; then kill ${CHILDREN} && wait; fi' TERM
+
+mkdir -p "${HOME}"
+
+# if the cluster profile included an insights secret, install it to the cluster to
+# report support data from the support-operator
+if [[ -f "${cluster_profile}/insights-live.yaml" ]]; then
+    oc create -f "${cluster_profile}/insights-live.yaml" || true
+fi
+
+# set up cloud-provider-specific env vars
+KUBE_SSH_BASTION="$( oc --insecure-skip-tls-verify get node -l node-role.kubernetes.io/master -o 'jsonpath={.items[0].status.addresses[?(@.type=="ExternalIP")].address}' ):22"
```

Comment:
Adding notes here so they're captured (as I'm reviewing the step registry)

This is wrong, bastion is an optional requirement for some steps, not required for all.

Reply:
In fact, we should probably move all "SSH requiring" things out to their own step.  It might even be appropriate for that always to be expecting the bastion.  Will have to think about that.